examples/3DTilesWireframe.html

```diff
@@ -62,7 +70,7 @@
         for (const tilesManager of frame3DPlanar.layerManager.tilesManagers)
           tilesManager.addEventListener(
             udvizBrowser.TilesManager.EVENT_TILE_LOADED,
-            udvizBrowser.appendWireframeToTileset
+            udvizBrowser.THREEUtil.appendWireframeToTileset
```

Comment:
Since this is a three.js contextual function tileset should be replace by something like Object3D